packages/helper-grammar-regex-collection/test.js

```diff
@@ -400,6 +400,18 @@ const fixtures = {
       'import com.company..myapp',
     ],
   },
+  NET_PROJ_PACKAGE: {
+    valid: [
+      '<PackageReference Include="EntityFramework" Version="6.2.0" />',
+      '< PackageReference Include="EntityFramework" Version="6.2.0" />',
+      '<PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="$(MicrosoftExtensionsCachingMemoryPackageVersion)" />',
+      '<PackageReference Include="Microsoft.AspNetCore.Cors">\n<Version>2.0.0</Version>\n</PackageReference>',
```

Comment:
Is the order of the attributes always the same or should be the Regex also allow  `<PackageReference Version="6.2.0" Include="EntityFramework" />`

Reply:
Good point, I've adjusted the regex so order doesn't matter and added a test for it